packages/govuk-frontend/src/govuk/components/accordion/accordion.puppeteer.test.js

```diff
@@ -712,148 +712,160 @@ describe('/components/accordion', () => {
             )
           })
         })
+      })
 
-        describe('errors at instantiation', () => {
-          let examples
+      describe('errors at instantiation', () => {
```

Comment:
Noticed we accidentally had our `errors at instantiation` wrapped within the `localisation` block.